### PR TITLE
Move approver-policy install/confiugration from usage to projects

### PR DESCRIPTION
**Pull request by JoshVanL**
As [per](https://github.com/cert-manager/website/pull/997#issuecomment-1148447620), this PR moves the approver-policy page into the new `/projects` section. The usages page has been replaced with a small blurb and a link to the projects page.

Also adds a bullet point to the usages `README.md` for approver-policy.

/assign @wallrj

### CODE REVIEW

File: content/docs/projects/README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ functionality, and complement the core cert-manager feature-set.
 
 These tools help with security, compliance and control.
 
+- [Apply Policy to cert-manager Certificates](./approver-policy.md):
+  [approver-policy](https://github.com/cert-manager/policy-approver) is a
+  cert-manager
+  [approver](https://cert-manager.io/docs/concepts/certificaterequest/#approval)
+  that will approve or deny CertificateRequests based on CRD defined policies.
```
Comment (wallrj) [inline content/docs/projects/README.md:16]: I find it confusing to have the link back to https://github.com/cert-manager/policy-approver here. 
```suggestion
- [approver-policy: ](./approver-policy.md):
  a cert-manager **approver** that will approve or deny CertificateRequests based on CRD defined policies.
```

I suggest having a single link here to the new page on cert-manager.io. And in there we can mention the link to the source code and link to the approver concepts.

File: content/docs/projects/README.md [hunk 2]
```diff
@@ -9,6 +9,11 @@ functionality, and complement the core cert-manager feature-set.
...
 - [Distributing Trust Bundles in Kubernetes](./trust.md): Using the trust
     operator to distribute trust bundles, like CA certificates, across a
     Kubernetes cluster.
```
Comment (wallrj) [inline content/docs/projects/README.md:19]: Similarly: 

```suggestion
- [trust](./trust.md): an 
    operator to distribute trust bundles, like CA certificates, across a
    Kubernetes cluster.
```

File: content/docs/usage/approver-policy.md [hunk 3]
```diff
@@ -3,322 +3,12 @@ title: Policy for cert-manager certificates
...
-    my-plugin:
-      values:
-        val-1: key-1
-```
+cert-manager [CertificateRequests](../concepts/certificaterequest/) can be
+rejected from being signed by using the [approval
+API](../concepts/certificaterequest/#approval).
+[approver-policy](https://github.com/cert-manager/approver-policy) is a
+cert-manager project that enables you to write policy to automatically manage
+this approval mechanism.
```
Comment (wallrj) [inline content/docs/usage/approver-policy.md:11]: I suggest not linking to the GitHub project here. We want to direct users to the cert-manager.io/projects page if that is to be the new home of the documentation.
```suggestion
approver-policy is a
cert-manager project that enables you to write policy to automatically manage
this approval mechanism.
```

**Review by wallrj — APPROVED** [review]
I like the new projects menu.
Just a couple of suggestions.

/lgtm
/hold in case you want to make the suggested changes.

**Comment by JoshVanL** [comment]
Thanks @wallrj, I have updated. PTAL

File: content/docs/projects/README.md [hunk 4]
```diff
@@ -9,6 +9,9 @@ functionality, and complement the core cert-manager feature-set.
 
 These tools help with security, compliance and control.
 
-- [Distributing Trust Bundles in Kubernetes](./trust.md): Using the trust
-    operator to distribute trust bundles, like CA certificates, across a
-    Kubernetes cluster.
+- [approver-policy: ](./approver-policy.md):
+  a cert-manager **approver** that will automatically approve or deny
+  certificate requests based on defined policy.
```
Comment (wallrj) [inline content/docs/projects/README.md:14]:
```suggestion
- [approver-policy](./approver-policy.md):
  a cert-manager **approver** that will automatically approve or deny
  certificate requests based on defined policy.
```

**Review by wallrj — APPROVED** [review]
Just one tiny nit.

/lgtm
/hold in case you want to also fix the nit

**Review by jahrlin — APPROVED** [review]
Great stuff, very accessible.

/lgtm

**Review by jakexks — COMMENTED** [review]
/lgtm

**Pull request merged**